internal/cli/server/server.go

```diff
@@ -94,6 +94,8 @@ func NewHandler(logger logr.Logger) (*gin.Engine, error) {
 	router.GET("/ready", func(c *gin.Context) {
 		c.JSON(http.StatusOK, gin.H{})
 	})
+	// No authentication, no logging, no session. This is epinio's version and auth information.
```

Comment:
Why no logging, or no version? The healtcheck is skipping the logs to avoid cluttering since is pinged every X seconds to get the status of the server. But the `info` endpoint is a valid API, and we should use the logger middleware and the version middleware.

Reply:
Fair points. Will work on it.

Reply:
done. commit b26a271778e8f27050ffb3d1fcbedeafd3249806